integrationtests/wildfly-modules/src/test/resources/infinispan-custom.xml

```diff
@@ -1,17 +1,17 @@
 <infinispan
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="urn:infinispan:config:10.1 https://infinispan.org/schemas/infinispan-config-10.1.xsd
-                            urn:infinispan:server:10.1 https://infinispan.org/schemas/infinispan-server-10.1.xsd"
-        xmlns="urn:infinispan:config:10.1"
-        xmlns:server="urn:infinispan:server:10.1">
+        xsi:schemaLocation="urn:infinispan:config:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-config-${infinispan.core.schema.version}.xsd
+                            urn:infinispan:server:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-server-${infinispan.core.schema.version}.xsd"
+        xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
+        xmlns:server="urn:infinispan:server:${infinispan.core.schema.version}">
 
     <cache-container default-cache="default" statistics="true">
         <transport cluster="${infinispan.cluster.name}" stack="${infinispan.cluster.stack:tcp}"/>
         <global-state />
         <local-cache name="default" />
     </cache-container>
 
-    <server xmlns="urn:infinispan:server:10.1">
+    <server xmlns="urn:infinispan:server:11.0">
```

Comment:
Shouldn't you be using `${infinispan.core.schema.version}`?

Reply:
Done